src/platform/testing/local-dev-mock-api/common.js

```diff
@@ -77,6 +77,7 @@ const responses = {
         { name: 'vaOnlineSchedulingFlatFacilityPage', value: true },
         { name: 'vaOnlineSchedulingProviderSelection', value: true },
         { name: 'vaOnlineSchedulingCheetah', value: true },
+        { name: 'vaOnlineSchedulingHomepageRefresh', value: true },
```

Comment:
@department-of-veterans-affairs/frontend-review-group This is the platform change, just to add a new feature flag to the default mock data.